### PR TITLE
Перов - Лабораторная работа #2

### PR DESCRIPTION
**Pull request by diper1998**

### CODE REVIEW

**Comment by diper1998** [comment]
@mezotaken @ivanvikhrev @Agricola87 Посмотрите, пожалуйста

**Comment by mezotaken** [comment]
:+1:

**Comment by ivanvikhrev** [comment]
:+1:

File: modules/interpolation-search/include/ItrSearch.h [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#include<string>
+
+class ItrSearch {
+ protected:
+    int* ptrData;
+    unsigned sizeData;
+    int64_t findIndex;
+
+ public:
+    ItrSearch(int* myPtrSortedData, unsigned mySizeData);
```
Comment (Agricola87) [inline modules/interpolation-search/include/ItrSearch.h]: For interface best to use verbose names, like sortedDataSizePointer

File: modules/interpolation-search/include/ItrSearch.h [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    ItrSearch();
+    ~ItrSearch();
+    int64_t GetFindIndex();
+    unsigned GetSizeData();
```
Comment (Agricola87) [inline modules/interpolation-search/include/ItrSearch.h]: For cpp better to use small case for function name, excluding constructor/destructor

Comment (alexbatashev) [reply modules/interpolation-search/include/ItrSearch.h]: @Agricola87 wrong. We follow Google C++ Coding Standards https://google.github.io/styleguide/cppguide.html#Function_Names

File: modules/interpolation-search/include/ItrSearch.h (hunk 1 above)
Comment (diper1998) [reply modules/interpolation-search/include/ItrSearch.h]: Исправил!

File: modules/interpolation-search/include/ItrSearch.h [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+#include<string>
+
+class ItrSearch {
+ protected:
+    int* ptrSortedData;
+    unsigned sizeSortedData;
+    int64_t findIndex;
+
+ public:
+    ItrSearch(int* myPtrSortedData, unsigned mySizeSortedData);
```
Comment (Agricola87) [inline modules/interpolation-search/include/ItrSearch.h]: More explicit function and argument name (user can see it in IDE and must clearly understand the sense)

Comment (diper1998) [reply modules/interpolation-search/include/ItrSearch.h]: Поправил!

**Pull request merged**